ctc_lib/config_command_test.go

```diff
@@ -0,0 +1,101 @@
+/*
+Copyright 2018 Google Inc. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package ctc_lib
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"reflect"
+	"testing"
+
+	"github.com/spf13/cobra"
+)
+
+func TestConfigCommandGet(t *testing.T) {
+	var testConfigCommand = ContainerToolCommand{
+		ContainerToolCommandBase: &ContainerToolCommandBase{
+			Command: &cobra.Command{
+				Use: "Hello Command",
+			},
+			Phase: "test",
+		},
+		Output: nil,
+		RunO: func(command *cobra.Command, args []string) (interface{}, error) {
+			return nil, nil
+		},
+	}
+
+	Version = "1.0.1"
+	ConfigFile = "testdata/testConfig.json"
```

Comment:
is this used in the test?

Reply:
These are Package Variables that Container Command Tool Lib Uses. 
These Package Variables are defined at https://github.com/GoogleCloudPlatform/runtimes-common/blob/master/ctc_lib/ctc_lib.go#L33

An Example setting these values is in Demos.
https://github.com/GoogleCloudPlatform/runtimes-common/blob/master/demo/ctc/cmd/root.go#L54

Reply:
ah, makes sense.  thanks.